modules/rules_cuda/0.1.2/MODULE.bazel

```diff
@@ -0,0 +1,17 @@
+module(
+    name = "rules_cuda",
+    compatibility_level = 1,
+    version = "0.1.2",
+)
+
+bazel_dep(name = "bazel_skylib", version = "1.2.1")
+bazel_dep(name = "platforms", version = "0.0.4")
+
+toolchain = use_extension("@rules_cuda//cuda:extensions.bzl", "local_toolchain")
```

Comment:
I would recommend moving this extension into `private` if possible. You register the toolchain below, so users don't have to (and shouldn't do it) themselves. That way you reduce your public Bzlmod API surface.

Reply:
Actually you make a good point about this and actually we probably want to expose to users the ability to register a custom path to their cuda install so I've put together this PR to fix that in rules_cuda: https://github.com/bazel-contrib/rules_cuda/pull/66